_custom_redirects

```diff
@@ -0,0 +1 @@
+/*  /.netlify/functions/server  200
```

Comment:
TODO: Document this in the readme and add a comment in this file mentioning you can add your own rewrite rules here and/or in the netlify.toml